toolchain/check/pattern_match.h

```diff
@@ -11,14 +11,13 @@
 namespace Carbon::Check {
 
 // The pattern-match counterparts of the patterns passed to CalleePatternMatch.
+// FIXME rename, recomment.
```

Comment:
Was this missed?

```suggestion
```

Reply:
Oh, whoops, yes.

I've updated the comment, but if it's OK with you I think it'll be simpler to keep the name unchanged for now, even though it's not a great fit, because the next PR is going to eliminate this struct altogether.

Reply:
That's fine, I just think we prefer TODO for anything being checked in, and when I see FIXME I read it as "meant to fix prior to review"

Reply:
Oh, absolutely, the FIXME was just an oversight.